tests/e2e/specs/plugin-activation.test.js

```diff
@@ -34,37 +34,87 @@ const deactivateSiteKit = async () => {
 };
 
 describe( 'Plugin Activation Notice', () => {
-	beforeEach( async () => {
-		await deactivateSiteKit();
-		await activatePlugin( 'e2e-tests-gcp-credentials-plugin' );
-	} );
+	describe( 'When Javascript is enabled', () => {
+		beforeEach( async () => {
+			await deactivateSiteKit();
+			await activatePlugin( 'e2e-tests-gcp-credentials-plugin' );
+		} );
 
-	afterEach( async () => {
-		await deactivatePlugin( 'e2e-tests-gcp-credentials-plugin' );
-		await activateSiteKit();
-	} );
+		afterEach( async () => {
+			await deactivatePlugin( 'e2e-tests-gcp-credentials-plugin' );
+			await activateSiteKit();
+		} );
+
+		it( 'Should be displayed', async () => {
+			await activateSiteKit();
+
+			await page.waitForSelector( '.googlesitekit-activation__title' );
+
+			await expect( page ).toMatchElement( 'h3.googlesitekit-activation__title', { text: /Congratulations, the Site Kit plugin is now activated./i } );
```

Comment:
I know it's unrelated, but this is probably a fine time to remove the element tags from these selectors and just stick with class names. It makes the tests a bit less annoying if we change tags and I don't think the intent here is to check which tag we're using 😄. @aaemnnosttv pointed this out to me in some tests I wrote and I agreed, so we might as well do it here too 😄
```suggestion
			await expect( page ).toMatchElement( '.googlesitekit-activation__title', { text: /Congratulations, the Site Kit plugin is now activated./i } );
```